source/indexes/create-vector-search-index.txt

```diff
@@ -0,0 +1,142 @@
+.. _compass-create-vector-search:
+
+===================================
+Create an Atlas Vector Search Index
+===================================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. versionadded:: 1.42.1
+
+You can create Atlas Vector Search indexes using |compass|. These indexes 
+enable you to index vector data *and* other data types, facilitating 
+semantic searches on the indexed fields. Atlas Vector Search indexes support 
+indexing vector data by identifying the most similar vectors. This is achieved 
+by calculating the distance between the query vector and the vectors stored 
+in the index.
+
+About this Task
+---------------
+
+To create an Atlas Search index, your deployment must be either:
+
+- Hosted on `MongoDB Atlas <https://www.mongodb.com/docs/atlas/>`__ and
+  have an Atlas cluster tier of M10 or higher.
+
+- A local deployment that is set up using the `Atlas CLI
+  <https://www.mongodb.com/docs/atlas/cli/stable/atlas-cli-deploy-local/>`__.
```

Comment:
[suggestion] for minimalism

```suggestion
- A local deployment that uses the `Atlas CLI
  <https://www.mongodb.com/docs/atlas/cli/stable/atlas-cli-deploy-local/>`__.
```

Reply:
In this case, we want to point out that it has to be "set up" with Atlas CLI so I may go ahead and keep this in.